content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

```diff
@@ -15,33 +15,20 @@
 
 <div class="layout" id="top">
 
-    <main class="content katacoda-content">
-
-        <div class="row">
-            <div class="col-md-12">
-                <p>
-                    पॉड कुबेरनेट्स एप्लिकेशन की मूल निष्पादन इकाई है। प्रत्येक पॉड आपके क्लस्टर पर चल रहे कार्यभार के एक हिस्से का प्रतिनिधित्व करता है। <a href="/docs/concepts/workloads/pods/">पॉड के बारे में अधिक जानें</a>।
-                </p>
-            </div>
-        </div>
-
-        <br>
-        <div class="katacoda">
-            <div class="katacoda__alert">
-                टर्मिनल का उपयोग करने के लिए, कृपया डेस्कटॉप या टैबलेट संस्करण का उपयोग करें
-            </div>
-
-            <div class="katacoda__box" id="inline-terminal-1"  data-katacoda-id="kubernetes-bootcamp/7" data-katacoda-color="326de6" data-katacoda-secondary="273d6d" data-katacoda-hideintro="false" data-katacoda-prompt="Kubernetes Bootcamp Terminal" style="height: 600px;">
-            </div>
-
-        </div>
-        <div class="row">
-            <div class="col-md-12">
-            	<a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/" role="button"> &lt; मॉड्यूल 1 पर लौटें<span class=""></span></a>
-            	<a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/" role="button">होम<span class=""></span></a>
-                <a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/explore/explore-intro/" role="button">मॉड्यूल 3 पर जाए &gt;<span class=""></span></a>
-            </div>
+    <main class="content">
+
+        <div>
+            <h2 class="katacoda-gone">
+                प्रलेखन अनुपलब्ध
+            </h2>
+            <p>
+                क्लस्टर बनाने के तरीके पर इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```

Comment:
Better to localize these in correct contexts OR we could do this
```suggestion
इंटरैक्टिव ट्यूटोरियल उपलब्ध नहीं है।
```